src/main/java/entities/User.java

```diff
@@ -29,13 +26,11 @@ public class User {
     *
     * @param admin whether the user will have admin priviledges
     * */
-    public User(boolean admin, int numUsersCreated, String email, String password) {
+    public User(boolean admin, int id, String email, String password) {
         this.favourites = new ArrayList<Post>();
         this.posts = new ArrayList<Post>();
```

Comment:
this should be an arraylist of integers (the ids of the posts), please also change the attributes as well thanks